lnd.go

```diff
@@ -721,3 +731,21 @@ func waitForWalletPassword(grpcEndpoint, restEndpoint string,
 		return nil, nil, fmt.Errorf("shutting down")
 	}
 }
+
+// parseHexColor takes a hex string representation of a color in the
+// form "#RRGGBB", parses the hex color values, and returns a color.RGBA
+// struct of the same color
+func parseHexColor(colorStr string) (color.RGBA, error) {
+	if len(colorStr) != 7 {
```

Comment:
This is cool, I would probably also ensure the first character is `#`, and then return an error like `Color must be in format "#RRGGBB"`